src/transformations.cpp

```diff
@@ -351,6 +352,8 @@ XPtrImage magick_image_annotate( XPtrImage input, Rcpp::CharacterVector text, co
   draw.push_back(Magick::DrawableTextAntialias(true));
   if(strokecolor.size())
     draw.push_back(Magick::DrawableStrokeColor(Color(strokecolor[0])));
+  if(strokewidth != 0)
```

Comment:
Does NULL (the default) really become 0 when the R integer vector is coerced to a double?

Reply:
I don't know.  I believe the default in magick is 1 and I've set the default in `image_annotate` to 1.  I haven't tested behavior if 0 is allowed. I haven't tested behavior if strokecolor isn't set but strokewidth is set either. So maybe some additional checks may be necessary; I don't have much familiarity with magick. I assume non-integer values may be non-sense but then I wasn't sure whether it's the job of this package to catch that.